src/main/java/io/jenkins/plugins/safebatch/BatchSanitizerGlobalRule.java

```diff
@@ -58,12 +57,12 @@
 public class BatchSanitizerGlobalRule implements EnvVarsFilterGlobalRule {
     private static final Logger LOGGER = Logger.getLogger(BatchSanitizerGlobalRule.class.getName());
 
-    // Ideally we would figure out a better way to filter safe/unsafe env vars in batch than just presence of specific characters
+    // Ideally we would figure out a better way to filter safe/unsafe env vars in batch than just presence of specific
+    // characters
     // Unfortunately, https://ss64.com/nt/syntax-esc.html#escape seems too complex without knowing the script
     @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "Accessible via System Groovy Scripts")
-    private static String DANGEROUS_CHARACTERS = System.getProperty(
-            BatchSanitizerGlobalRule.class.getName() + ".DANGEROUS_CHARACTERS",
-            "|^&%\"<>");
+    private static String DANGEROUS_CHARACTERS =
```

Comment:
`DANGEROUS_CHARACTERS` is private, there is no need for the Suppress annotation, SpotBugs is complaining about it:
```
[ERROR] Medium: Suppressing annotation on the field io.jenkins.plugins.safebatch.BatchSanitizerGlobalRule.DANGEROUS_CHARACTERS is unnecessary [io.jenkins.plugins.safebatch.BatchSanitizerGlobalRule] In BatchSanitizerGlobalRule.java US_USELESS_SUPPRESSION_ON_FIELD
```